packages/connector-types/src/index.ts

```diff
@@ -0,0 +1,107 @@
+import { Languages } from '@logto/phrases';
```

Comment:
```suggestion
import type { Languages } from '@logto/phrases';
```

this can just import type from the package